java/flipkartPackage/FlipKartWithOutLogin.java

```diff
@@ -0,0 +1,279 @@
+
+import java.awt.Point;
+import java.awt.RenderingHints.Key;
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import io.appium.java_client.AppiumBy;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.nativekey.AndroidKey;
+import io.appium.java_client.android.nativekey.KeyEvent;
+import io.appium.java_client.android.options.UiAutomator2Options;
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+import io.appium.java_client.service.local.AppiumServiceBuilder;
+
+public class FlipKartWithOutLogin {
+
+	public AppiumDriverLocalService localservices;
+
+	public AppiumServiceBuilder service;
+
+	public UiAutomator2Options capabilities;
+
+	public AndroidDriver driver;
+
+	public ChangeAddress change;
+
+	public WebDriverWait wait;
+
+	public MobileGestures gest;
+
+	@BeforeTest
+	public void startService() throws MalformedURLException, URISyntaxException {
+		// Starting the Appium
+		// Server=====================================================================
+
+		service = new AppiumServiceBuilder();
+
+		service.withAppiumJS(
+				new File("C:\\Users\\ADMIN\\AppData\\Roaming\\npm\\node_modules\\appium\\build\\lib\\main.js"))
+				.withIPAddress("127.0.0.1").withTimeout(Duration.ofSeconds(20)).usingPort(4723).build();
+
+		localservices = AppiumDriverLocalService.buildService(service);
+
+		localservices.start();
+
+		// Setting the
+		// capabilities=============================================================================
+
+		capabilities = new UiAutomator2Options();
+
+		capabilities.setDeviceName("Redmi");
+
+		capabilities.setPlatformName("Android");
+
+		capabilities.setPlatformVersion("10.0");
+
+		capabilities.setAutomationName("UiAutomator2");
+
+		capabilities.setAppPackage("com.flipkart.android");
+
+		capabilities.setAppActivity("com.flipkart.android.activity.HomeFragmentHolderActivity");
+
+		// capabilities.setCapability("adbExecTimeout", "20000");
+
+		capabilities.setNoReset(true);
+
+		// capabilities.setApp(
+		// "C:\\Users\\ADMIN\\NavaDhitiWorkSpace\\AppiumFlipKart\\src\\test\\java\\utils\\com.flipkart.android_8.0.apk");
+
+		driver = new AndroidDriver(new URI("http://127.0.0.1:4723/").toURL(), capabilities);
+
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+	}
+
+	@Test
+	public void flipKartAutomation() throws InterruptedException {
+
+		gest = new MobileGestures(driver);
+
+		WebElement account = driver.findElement(By.xpath("//android.widget.TextView[@text=\"Account\"]"));
```

Comment:
make this simple for example 

driver.findElement(By.xpath("//android.widget.TextView[@text=\"Account\"]")).click();